katas/content/linear_algebra/index.md

```diff
@@ -145,14 +145,14 @@ Scalar multiplication has the following properties:
     "title": "Matrix Multiplication" 
 })
 
-**Matrix multiplication** is a very important and somewhat unusual operation. The unusual thing about it is that neither its operands nor its output are the same size: an $n \times m$ matrix multiplied by an $m \times k$ matrix results in an $n \times k$ matrix. 
+**Matrix multiplication** is a very important and somewhat unusual operation. The unusual thing about it's that neither its operands nor its output are the same size: an $n \times m$ matrix multiplied by an $m \times k$ matrix results in an $n \times k$ matrix. 
```

Comment:
This is a very odd replacement. It seems much more correct before converting `it is` to `it's`.

Reply:
My mistake, I used the replace option and didn't read it correctly